### DISCUSSION TITLE
Is there a way to manually change the print time estimate?

### DISCUSSION THREAD

**Discussion by Boozebeard** [sn4k3/UVtools#553] [Q&A]
The print estimate is really off for my Phrozen Sonic Mighty 8k for what ever reason. Is there a way to manually override the per layer time estimate like in lychee? Or is there some way I can get the estimate to be more accurate for my printer?

**Comment by sn4k3** [comment]
Time is calculated based on instant changes in speed and so the slower acceleration on your printer the more off time will be.
Currently there is no way to change error in calculation

**Reply by Boozebeard** [reply]
Thanks for clearing that up. Would be a nice feature to add :)

**Reply by sn4k3** [reply]
Proper calculation would be with acceleration values in place. I will take a look.
